nginx-mixin/dashboards/nginx-metrics.json

```diff
@@ -0,0 +1,567 @@
+{
+  "__inputs": [
+    {
+      "description": "",
+      "label": "Prometheus",
+      "name": "DS_PROMETHEUS",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus",
+      "type": "datasource"
+    }
+  ],
+  "__requires": [
+    {
+      "id": "grafana",
+      "name": "Grafana",
+      "type": "grafana",
+      "version": "5.0.0"
+    },
+    {
+      "id": "graph",
+      "name": "Graph",
+      "type": "panel",
+      "version": ""
+    },
+    {
+      "id": "prometheus",
+      "name": "Prometheus",
+      "type": "datasource",
+      "version": "1.0.0"
+    },
+    {
+      "id": "singlestat",
```

Comment:
To my point above, singlestat panels aren't supported at all in current versions of Grafana Cloud for example, instead using stat panels.

Could you load this into a cloud instance, hit save dashboard, and then replace all the instances of: `"uid": "grafanacloud-prom"` datasource values with `"uid": "$(datasource)"`